pkg/volume/local/local_test.go

```diff
@@ -76,6 +76,9 @@ func getPersistentPlugin(t *testing.T) (string, volume.PersistentVolumePlugin) {
 }
 
 func getTestVolume(readOnly bool, path string) *volume.Spec {
+	if err := os.MkdirAll(path, 0777); err != nil {
+		return nil
```

Comment:
Function calling getTestVolume is not inspecting for err so I thought by return nil, will force calling function to fail and generate an error code. Would it be ok, otherwise I will need to refactor each call to add error handling. Please let me know.